install.sh

```diff
@@ -330,22 +346,61 @@ __add_zeek_to_path() {
     _ZEEK_IN_PATH=true
 }
 
+__mongo_upgrade_info() {
+    printf "$_IMPORTANT Mongo is already installed and is version $_MONGO_INSTALLED_VERSION.\n"
+    printf "$_IMPORTANT This will upgrade Mongo to version $_MONGO_VERSION.\n"
+    printf "$_IMPORTANT Note that Mongo must be upgraded to $_MONGO_VERSION for RITA $_RITA_VERSION to work.\n"
+    printf "$_IMPORTANT We suggest creating a backup of your data before upgrading (https://docs.mongodb.com/manual/tutorial/backup-and-restore-tools/).\n"
+    printf "$_QUESTION Would you like to upgrade your Mongo instance [y/N] "
+    read -e
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+        exit 0
+    fi
+}
+
+__intermediary_update_mongodb() {
+    # Need to stop mongo before updating otherwise we can end up with weird issues,
+    # such as the console version not matching the server version
+    systemctl is-active --quiet mongod && systemctl stop mongod
+
+    __install_mongodb "$_MONGO_MIN_UPDATE_VERSION"
+
+    # Need to also install all the components of the mongodb-org metapackage for Ubuntu
+    __install_packages mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools
+
+    if [ "$_MONGO_INSTALLED" = "true" ]; then
+        # Star and configure the service so that we can run the command to update feature compatibility
+        __configure_mongodb
+
+        # Wait for service to come to life
+        sleep 10
+
+        # Need to update feature compatibility to 4.0 otherwise things will break when we update
+        # to 4.2
+        __load "$ITEM Updating feature comapibility in Mongo to 4.0" __update_feature_compatibility
+    fi
+}
+
+__update_feature_compatibility() {
```

Comment:
Yep, I can give it a whirl. I hardcoded because I wasn't sure on the proper escaping there and didn't want to deal with it while debugging. I will get that fixed up!